test/torch/model/test_jit.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+import pytest
+
+import torch
+
+from gluonts.torch.model.deepar import DeepARModel
+from gluonts.torch.model.tft import TemporalFusionTransformerModel
+
+
+def get_model_and_input(model):
```

Comment:
What's the purpose of this, why not just use `.example_input_array`?

Also, why should this be part of the parameter and not just be invoked in the test function?

Reply:
I'm not sure that using these completely dummy arrays is the best approach. Maybe for tracing the model yes, but then we should probably test the model vs scripted model using some other input

Reply:
So it's mostly a placeholder for now

Reply:
> What's the purpose of this, why not just use .example_input_array?

That's not a property of `torch.nn.Module`

Reply:
Right. Still looks like something which could be consolidated.